(t) Missing serializers `fields` description in RockOnDeviceSerializer

This issue was originally reported with a fix by forum user `@frier`:
[https://forum.rockstor.com/t/unknown-client-error-doing-a-get-to-api-rockons-devices-152/7661/13](https://forum.rockstor.com/t/unknown-client-error-doing-a-get-to-api-rockons-devices-152/7661/13?u=flox)

The install of a Rock-On using the `DContainerDevice` model fails due to a missing definition in our serializers. Briefly, our recent Django update (see #2365) required further modifications in our serializers but it appears the `RockOnDeviceSerializer` did not receive the required updates.
Forum user `@frier` fortunately identified a fix:
the following:
https://github.com/rockstor/rockstor-core/blob/f27b1e09fd68f5ffc9ded4d0790e6e8b3c1da8ad/src/rockstor/storageadmin/serializers.py#L330-L332

should read:
```python
class RockOnDeviceSerializer(serializers.ModelSerializer):
    class Meta:
        model = DContainerDevice
        fields = "__all__"
```

This fix was later confirmed to work by forum user `@Jorma_Tuomainen`.

Reproducer:
Click "Install" on any Rock-on that has an associated entry in the `DContainerDevice` model: let's use "Emby server" as example. Other Rock-Ons NOT depending on that model are not affected by this bug.

In the webUI, the Rock-On install wizard modal shows up, but remains blank.
In `rockstor.log`, we see the following:
```
[04/Jan/2023 14:51:57] ERROR [storageadmin.middleware:33] Exception occurred while processing a request. Path: /api/rockons/devices/46 method: GET
[04/Jan/2023 14:51:57] ERROR [storageadmin.middleware:34] (u"Creating a ModelSerializer without either the 'fields' attribute or the 'exclude' attribute has been deprecated since 3.3.0, and is now disallowed. Add an explicit fields = '__all__' to the RockOnDeviceSerializer serializer.",)
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/usr/local/lib/python2.7/site-packages/django/views/decorators/csrf.py", line 58, in wrapped_view
    return view_func(*args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/django/views/generic/base.py", line 68, in view
    return self.dispatch(request, *args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/rest_framework/views.py", line 495, in dispatch
    response = self.handle_exception(exc)
  File "/usr/local/lib/python2.7/site-packages/rest_framework/views.py", line 455, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/usr/local/lib/python2.7/site-packages/rest_framework/views.py", line 466, in raise_uncaught_exception
    raise exc
AssertionError: (u"Creating a ModelSerializer without either the 'fields' attribute or the 'exclude' attribute has been deprecated since 3.3.0, and is now disallowed. Add an explicit fields = '__all__' to the RockOnDeviceSerializer serializer.",)
```

--- a/src/rockstor/storageadmin/serializers.py
+++ b/src/rockstor/storageadmin/serializers.py
@@ -330,6 +330,7 @@ class RockOnEnvironmentSerializer(serializers.ModelSerializer):
 class RockOnDeviceSerializer(serializers.ModelSerializer):
     class Meta:
         model = DContainerDevice
+        fields = "__all__"
 
 
 class RockOnLabelSerializer(serializers.ModelSerializer):

